src/sentry/static/sentry/app/views/organizationIntegrationConfig.jsx

```diff
@@ -0,0 +1,305 @@
+import IndicatorStore from '../stores/indicatorStore';
+import LoadingError from '../components/loadingError';
+import Panel from './settings/components/panel';
+import PanelBody from './settings/components/panelBody';
+import PanelHeader from './settings/components/panelHeader';
+import PanelItem from './settings/components/panelItem';
+import PluginIcon from '../plugins/components/pluginIcon';
+import SettingsPageHeader from './settings/components/settingsPageHeader';
+import marked from '../utils/marked';
+
+const IntegrationIcon = styled.img`
+  height: 32px;
+  width: 32px;
+  border-radius: 3px;
+  display: block;
+`;
+
+const IntegrationName = styled.div`
+  font-size: 16px;
+  margin-bottom: 3px;
+`;
+
+const DomainName = styled.div`
+  color: ${p => p.theme.gray3};
+  font-size: 14px;
+`;
+
+const Details = styled(Flex)`
+  font-size: 15px;
+  line-height: 2.1rem;
+`;
+
+const Description = styled.div`
+  li {
+    margin-bottom: 6px;
+  }
+`;
+
+const AuthorName = styled.div`
+  color: ${p => p.theme.gray2};
+`;
+
+const MetadataLink = styled.a`
+  display: block;
+  margin-bottom: 6px;
+`;
+
+const alertLinkMarkedRenderer = new marked.Renderer();
+alertLinkMarkedRenderer.paragraph = s => s;
+
+function computeCenteredWindow(width, height) {
+  const screenLeft = window.screenLeft != undefined ? window.screenLeft : screen.left;
+  const screenTop = window.screenTop != undefined ? window.screenTop : screen.top;
+  const innerWidth = window.innerWidth
+    ? window.innerWidth
+    : document.documentElement.clientWidth
+      ? document.documentElement.clientWidth
+      : screen.width;
+
+  const innerHeight = window.innerHeight
+    ? window.innerHeight
+    : document.documentElement.clientHeight
+      ? document.documentElement.clientHeight
+      : screen.height;
+
+  const left = innerWidth / 2 - width / 2 + screenLeft;
+  const top = innerHeight / 2 - height / 2 + screenTop;
+
+  return {left, top};
+}
+
+export default class OrganizationIntegrationConfig extends AsyncView {
+  componentDidMount() {
+    this.dialog = null;
+    window.addEventListener('message', this.receiveMessage, false);
+  }
+
+  componentWillUnmount() {
+    super.componentWillUnmount();
+    window.removeEventListener('message', this.receiveMessage);
+
+    if (this.dialog !== null) {
+      this.dialog.close();
+    }
+  }
+
+  getProvider() {
+    const {config} = this.state;
+
+    if (config !== null) {
+      return config.providers.find(p => p.key == this.props.params.providerKey) || null;
+    }
+
+    return null;
+  }
+
+  getTitle() {
+    const provider = this.getProvider();
+
+    if (provider === null) {
+      return 'Global Integrations';
+    }
+
+    return `${provider.name} Integration`;
+  }
+
+  getEndpoints() {
+    const {orgId} = this.props.params;
+
+    return [
+      ['config', `/organizations/${orgId}/config/integrations/`],
+      ['itemList', `/organizations/${orgId}/integrations/`, {query: {status: ''}}],
```

Comment:
why does it need the empty query? worth commenting?

Reply:
Dunno, copied this from the other integrations page